core/app_modules/cal.js

```diff
@@ -21,6 +28,8 @@ module.exports.addCalendarEvent = co.wrap(function*(username, event, user){
   }
   if (!isOverlap) {
     yield client.nonQuery('INSERT INTO calendar("user", "time", room, duration, title)VALUES ($1, $2, $3, $4, $5);', [user, event.time, event.room, event.duration, event.title]);
+    user_info = yield get_user_info(user);
+    email.roomConfirmation(user_info, event.time, event.room)

```

Comment:
@Luke1298, I think if the user doesn't have an email address we should just fail quietly. It might be prudent to log the details of the error, but if someone doesn't get an email it's not the end of the world. They'll probably anticipate getting the room reserved anyway.

We probably want to be more explicit in our wording when people request room reservations as well, otherwise I'm afraid people with think they're reserving the room without it actually being the case. @alexcdahl, what do you think? Can we put a message on the requested room that says "Pending Something-orOther"?